FlagsmithClient/src/test/java/com/flagsmith/TraitsTests.kt

```diff
@@ -96,20 +96,46 @@ class TraitsTests {
         mockServer.mockResponseFor(MockEndpoint.SET_TRAITS)
         runBlocking {
             val result =
-                flagsmith.setTraitsSync(listOf(Trait(key = "set-from-client", value = "12345")), "person")
+                flagsmith.setTraitsSync(listOf(Trait(key = "set-from-client", traitValue = "12345")), "person")
             assertTrue(result.isSuccess)
             assertEquals("set-from-client", result.getOrThrow().first().key)
             assertEquals("12345", result.getOrThrow().first().stringValue)
             assertEquals("person", result.getOrThrow().first().identity.identifier)
         }
     }
 
+    @Test
+    fun testSetTraitsWithTransient() {
+        mockServer.mockResponseFor(MockEndpoint.SET_TRANSIENT_TRAITS)
+        runBlocking {
+            val result =
+                flagsmith.setTraitsSync(
+                    listOf(
+                        Trait(
+                            key = "trait-one-with-transient",
+                            traitValue = "transient-trait-one",
+                            transient = true
+                        ),
+                        Trait(
+                            key = "trait-two-with-transient",
+                            traitValue = "transient-trait-two",
```

Comment:
The keys / values here aren't very consistent. For example, `traitValue = "transient-trait-two"` then actually sets `transient = false` ?

Reply:
Corrected